.github/scripts/check_owners.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+# This script will check for an OWNERS file in all task and pipeline 
+# directories provided either via DIRECTORIES env var, or as 
+# arguments when running the script.
+#
+# Examples of usage:
+# export DIRECTORIES="mydir/tasks/apply-mapping some/other/dir"
+# ./check_owners.sh
+#
+# or
+#
+# ./check_owners.sh mydir/tasks/apply-mapping some/other/dir
+
+if [ $# -gt 0 ]; then
+  DIRECTORIES=$@
+fi
+
+if [ -z "${DIRECTORIES}" ]; then
+  echo Error: No directories as argument.
+  echo Usage:
+  echo "$0 [item1] [item2] [...]"
+  exit 1
+fi
+
+# check every item is a directory
+for DIR in $DIRECTORIES; do
+  if [[ -d "$DIR" ]]; then
+    true
+  else
+    echo "Error: Not a directory: $DIR"
+    exit 1
+  fi
+done
+
+TEAM_NAME=release-service-maintainers
+
+for DIR in $DIRECTORIES
```

Comment:
I think this is better, but (apologies for not mentioning it in the last review, I didn't notice) I think this loop can also be combined with the one started on line 27